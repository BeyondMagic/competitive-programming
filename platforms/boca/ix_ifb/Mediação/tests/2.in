3
7
2 6 9 4 7 1 9
5
1 2 3 4 5
4
12 4 9 3
