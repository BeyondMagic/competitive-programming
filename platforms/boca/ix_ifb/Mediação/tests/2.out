4
A cadeira, usa a cadeira!
2
